### DISCUSSION TITLE
@quasar/vite-plugin unbelievable slow

### DISCUSSION THREAD

**Discussion by ericrange** [quasarframework/quasar#17442] [General - Components / Directives / etc]
after we finally! figured out how to upgrade to quasar 2.16, we need to install and use "@quasar/vite-plugin", which was not necessary with quasar 2.15.

now, every quasar component needs 10 seconds to be rendered. why? also our ci performance is at least 10x slower than with 2.15.

is this to be expected? if not, how to fix that?

**Comment by ericrange** [comment]
even in production mode is my app soooo slow. i reverted back to 2.15.

am i the only one with this performance issues?

**Reply by ivanjaros** [reply]
yes 😀
i have recently found issue between 2.16.2 and 2.16.9, or it could have been vue 3.4.27 vs 3.4.38, not sure. i fixed it and moved on. but slowdown in components is something most likely related to some misconfiguration. i know there were some changes in how the quasar config is built. you can clearly see it in gitignore now. so i think it might be related to that. Best course of action will probably be to make a clean installation and simply copy your components and code back but walk the quasar config manually and make changes line by line to avoid this issue.
